tests/approaches/test_interactive_approach.py

```diff
@@ -55,18 +55,30 @@ def test_create_teacher_dataset():
 def test_interactive_learning_approach():
     """Test for InteractiveLearningApproach class, entire pipeline."""
     utils.update_config({
-        "env": "cover",
-        "approach": "interactive_learning",
-        "excluded_predicates": "",
-        "timeout": 10,
-        "max_samples_per_step": 10,
-        "seed": 12345,
-        "sampler_mlp_classifier_max_itr": 500,
-        "predicate_mlp_classifier_max_itr": 500,
-        "neural_gaus_regressor_max_itr": 500,
-        "interactive_num_episodes": 1,
-        "interactive_relearn_every": 1,
-        "interactive_known_predicates": {'HandEmpty', 'Holding', 'Covers'}
+        "env":
```

Comment:
lol oh gosh, did the autoformatter do this?

Reply:
I guess the autoformatter is boss now...

Reply:
yeah....